include/libm2k/digital/m2kdigital.hpp

```diff
@@ -423,6 +423,22 @@ class LIBM2K_API M2kDigital
 	 */
 	virtual void resetRateMux() = 0;
 
+	/**
+	 * @brief Set the clocksource
+	 * @param external - True to set source to external
+	 *		   - False to set source to internal
+	 *
+	 * @note Only available from firmware v0.26.
+	 */
+	virtual void setExternalClocksource(bool external) = 0;
+
+	/**
+	 * @brief Get the current clocksource
+	 * @return a string of the clocksource
+	 *
+	 * @note Only available from firmware v0.26.
+	 */
+	virtual std::string getClocksource() = 0;
```

Comment:
Because the setter uses boolean value, I suppose that the getter should work in the same way. Something like bool isExternClockSet().